jdaviz/configs/default/plugins/subset_plugin/subset_plugin.vue

```diff
@@ -83,7 +83,7 @@
           v-model.number="item.value"
           type="number"
           style="padding-top: 0px; margin-top: 0px"
-          :suffix="item.unit.replace('Angstrom', 'A')"
+          :suffix="item.unit ? item.unit.replace('Angstrom', 'A') : ''"
```

Comment:
in short: if `item.unit` exists, parse it and show the string (shortening "Angstrom" to "A" since unfortunately unicode is not supported, otherwise don’t show a unit at all.